gulp/docs.js

```diff
@@ -65,25 +62,31 @@ gulp.task('docs:scripts', () => {
   return gulp
     .src('docs/assets/javascript/application.mjs')
     .pipe(
-      esbuild({
-        outfile: 'application.js',
-        target: 'es6',
+      gulpEsbuild({
+        bundle: true,
+        loader: { '.mjs': 'js' },
```

Comment:
~I'm assuming this is correct, but just wanted to check it shouldn't be `.js`?~

Just re-read the line, and spotted the colon, and now understand it!